apps/rush-lib/src/utilities/Utilities.ts

```diff
@@ -517,20 +555,56 @@ export class Utilities {
     return new Error('Unable to find rush.json configuration file');
   }
 
+  private static _executeLifecycleCommandInternal<TCommandResult>(
+    command: string,
+    rushConfiguration: RushConfiguration | undefined,
+    spawnFunction: (command: String, args: string[], spawnOptions: child_process.SpawnOptions) => TCommandResult,
+    options: ILifecycleCommandOptions
+  ): TCommandResult {
+    let shellCommand: string = process.env.comspec || 'cmd';
+    let commandFlags: string = '/d /s /c';
+    let useShell: boolean = true;
+    if (process.platform !== 'win32') {
+      shellCommand = 'sh';
+      commandFlags = '-c';
+      useShell = false;
+    }
+
+    const environment: IEnvironment = Utilities._createEnvironmentForRushCommand(
+      {
+        initCwd: options.initCwd,
+        pathOptions: {
+          ...options.environmentPathOptions,
+          projectRoot: options.workingDirectory,
+          commonTempFolder: rushConfiguration ? rushConfiguration.commonTempFolder : undefined
```

Comment:
Before merging, we should test that `rushx` still works in a standalone project folder (i.e. without any **rush.json**).

Reply:
It works :)